server/app.ts

```diff
@@ -62,6 +63,7 @@ export default function createApp(
   app.use('/link', setUpRequestLink(magicLinkService))
   app.use('/link', setupLinkEmailSent())
   app.use('/link', setUpVerifyLink(magicLinkService))
+  app.use('/contact-helpdesk', setupContactHelpdesk())
```

Comment:
As discussed earlier, 2 routes for the contact helpdesk form - one for external users, one for mail room users, so that we can apply the correct title in the header!